packages/console/src/components/MultiTextInput/index.tsx

```diff
@@ -47,10 +58,15 @@ const MultiTextInput = ({ title, value, onChange, onKeyPress, error, placeholder
   };
 
   return (
-    <div className={styles.multilineInput}>
+    <div className={classNames(styles.multilineInput, className)}>
       {fields.map((fieldValue, fieldIndex) => (
-        // eslint-disable-next-line react/no-array-index-key
-        <div key={fieldIndex}>
+        <div
+          // eslint-disable-next-line react/no-array-index-key
+          key={fieldIndex}
```

Comment:
I don't think we should use indexes as keys, as the inputs might be deleted and the indexes would be changed

Reply:
This is a special case, for the value is `string[]`, and if any input is deleted, the whole component will be re-rendered.